pandas/core/base.py

```diff
@@ -656,6 +656,10 @@ class IndexOpsMixin:
     def transpose(self, *args, **kwargs):
         """
         Return the transpose, which is by definition self.
+
+        Returns
+        -------
+        self: %(klass)s
```

Comment:
I think having self here will cause other validation errors as this line is reserved only for type